### DISCUSSION TITLE
Image is not found after deployment

### DISCUSSION THREAD

**Discussion by talhabw** [nuxt/nuxt#18107] [Questions]
Hi. I started a project with Nuxt 3. I added my images to a _img_ under the _public_ folder. I am trying to access them by `src="/img/<filename>"` It works fine on development mode. But when I deploy it to Vercel, images are not here. 

You can take a look at my repo for reproduction and deployed vercel link:
https://github.com/talhabw/ealrobotics
https://ealrobotics.vercel.app/

How can I solve this problem? Thanks in advance.

**Comment by mihail727** [comment]
Exactly the same problem if the name of the image is written in Russian...
